conf/modules.config

```diff
@@ -37,8 +37,8 @@ process {
     }
 
     withName: 'BCFTOOLS_REHEADER' {
-        beforeScript = { "echo ${meta.label} > ${meta.label}.txt" }
-        ext.args     = { "--samples ${meta.label}.txt" }
+        errorStrategy = { task.attempt <= 3 ? 'retry' : 'terminate' }
+        maxRetries = 3
```

Comment:
Afaik resource management is done in `base.config`. Its just a little confusing when that is all over the place imho.